pkg/hintrunner/zero/zerohint_ec_test.go

```diff
@@ -0,0 +1,184 @@
+package zero
+
+import (
+	"testing"
+
+	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+)
+
+func TestZeroHintEc(t *testing.T) {
+	runHinterTests(t, map[string][]hintTestCase{
+		"EcNegate": {
+			{
```

Comment:
I think it would be way more readable for the reviewers, if we start labeling the testcases somehow. Maybe just a simple comment would be enough (maybe enforce via linter) or adding a new field to the test case with the label (but it would be redundant)

Reply:
Actually, for running tests we pass as the name of the test `<TestGroup>_<i>` ([here](https://github.com/NethermindEth/cairo-vm-go/blob/39ec9f675e028685085aec862fd459a58e935baa/pkg/hintrunner/zero/zerohint_test.go#L290)) where `TestGroup` would be the name of the hint you're testing and `i` the specific case being tested for that hint. I think this should be good enough for knowing which specific test is failing, but it's true we could get a more readable error output if adding a label instead of having to count which case it is.
I believe adding a comment at the top of each case explaining what you want to achieve with it is very helpful. In fact, it was the approach I was taking.
Adding a label to the tests sounds like a good idea, but having so many test cases to modify, it would be better to address this in a different PR. For now, anyway, could you add some comments?

wdyt?

Reply:
sounds good to me